profiles/fscloud/variables.tf

```diff
@@ -0,0 +1,125 @@
+variable "plan" {
+  type        = string
+  description = "Plan for the event stream instance. lite, standard or enterprise-3nodes-2tb"
+  default     = "enterprise-3nodes-2tb"
```

Comment:
I don't think lite or standard are fscloud compliant (you will have to check). If that is the case, you should remove this variable and hard code `enterprise-3nodes-2tb` in the fscloud profile main.tf

Reply:
Correct as per https://cloud.ibm.com/docs/EventStreams?topic=EventStreams-plan_choose#plan_enterprise

Reply:
done